docs/spec/callables.rst

```diff
@@ -522,31 +520,31 @@ and overloads. They can be defined as protocols with a ``__call__`` member::
       ...
 
   comb: Combiner = good_cb  # OK
-  comb = bad_cb  # Error! Argument 2 has incompatible type because of
+  comb = bad_cb  # Error! Argument 2 is not assignable because of
                  # different parameter name and kind in the callback
 
 Callback protocols and ``Callable[...]`` types can generally be used
 interchangeably.
 
 
-Subtyping rules for callables
------------------------------
+Assignability rules for callables
+---------------------------------
 
-A callable type ``A`` is a subtype of callable type ``B`` if the return type
-of ``A`` is a subtype of the return type of ``B`` and the input signature
-of ``A`` accepts all possible combinations of arguments that the input
-signature of ``B`` accepts. All of the specific subtyping rules described below
-derive from this general rule.
+A callable type ``B`` is :term:`assignable` to callable type ``A`` if the
+return type of ``B`` is assignable to the return type of ``A`` and the input
+signature of ``B`` accepts all possible combinations of arguments that the
+input signature of ``A`` accepts. All of the specific assignability rules
+described below derive from this general rule.
 
 
 Parameter types
 ^^^^^^^^^^^^^^^
 
 Callable types are covariant with respect to their return types but
 contravariant with respect to their parameter types. This means a callable
-``A`` is a subtype of callable ``B`` if the types of the parameters of
-``B`` are subtypes of the parameters of ``A``. For example,
-``(x: float) -> int`` is a subtype of ``(x: int) -> float``::
+``B`` is assignable to callable ``A`` if the types of the parameters of
```

Comment:
I'd link this term to the glossary definition a little more regularly throughout this document (probably at least once per subsection, personally). It's a very useful term, but it's probably not self-evident to newcomers what it means, and you can't necessarily expect people to have read previous subsections before they start on this subsection -- they might just skip to the subsection that's relevant

```suggestion
``B`` is :term:`assignable` to callable ``A`` if the types of the parameters of
```